app/controllers/items_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ItemsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @list = List.find(params[:list_id])
+    @user = current_user
+    @item = @list.items.build(item_params)
+    @item.list = @list
+    @new_item = Item.new
+
+
+    if @item.save
+      flash[:notice] = "Item was saved."
+      redirect_to user_list_path(@user, @list)
+    else
+      flash[:error] = "There was an error saving the item. Please try again."
+      render :new
```

Comment:
Something doesn't feel quite right about doing a `render :new` here along with the `@new_item` assignment at line 9. 

A redirect might be more appropriate. 

What's the ideal behaviour you want to happen when `@item.save` fails?